src/test/java/org/broadinstitute/hellbender/tools/PrintReadsCRAMIntegrationTest.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.hellbender.tools;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.read.SamAssertionUtils;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public final class PrintReadsCRAMIntegrationTest extends CommandLineProgramTest{
+
+    private static final File TEST_DATA_DIR = getTestDataDir();
+
+    @Override
+    public String getTestedClassName() {
+        return PrintReads.class.getSimpleName();
+    }
+
+
+    @Test(dataProvider="testingData")
+    public void testFileToFile(String fileIn, String extOut) throws Exception {
+        String samFile= fileIn;
+        final File outFile = File.createTempFile(samFile + ".", extOut);
+        outFile.deleteOnExit();
+        File ORIG_BAM = new File(TEST_DATA_DIR, samFile);
+        File reference = new File(TEST_DATA_DIR, "print_reads.fasta");
```

Comment:
Have the reference passed in as a parameter from the `DataProvider` (just in case we one day want to add test cases that require a different reference)